docs/sources/developers/plugins/extend-the-grafana-ui-with-links-and-commands.md

```diff
@@ -0,0 +1,106 @@
+# Extend the Grafana UI with links and commands
+
+Use the Plugin extensions API with your Grafana App plugins to add links to the Grafana UI. Doing so allows you to direct users to your plugins pages from various "placements" within the Grafana application.
+
+For a plugin to successfully register links it must:
+
+- Be an App plugin.
+- Be preloaded.
+- Be installed and enabled.
+
+## Add a link extension within Grafana
+
+Here's how you can add a link to the dashboard panel menus in Grafana via your plugin:
+
+Define the link extension in your plugin's `module.ts` file. First, define a new instance of the `AppPlugin` class by using the `configureExtensionLink` method. This method takes an object that describes your link extension, including a `title` property for the link text, a `placement` that tells Grafana where the link should appear, and a `path` for the user to navigate to your plugin.
+
+```typescript
+new AppPlugin().configureExtensionLink({
+  title: 'Go to basic app',
+  description: 'Will navigate the user to the basic app',
+  placement: 'grafana/dashboard/panel/menu',
```

Comment:
similar comment here, what are valid placements?

Reply:
I guess anything that starts with either `grafana/...` or `plugins/<plugin-id>/...`